### PR TITLE
add CausalEffectsMatrix in SCM

### PR DESCRIPTION
**Pull request by JorgeLuizFranco**
We want to given a vector of features(where number of features = number of nodes in the causal graph) $`x`$ create a decoded vector $`x_{\text{decoded}}`$ that has new values corresponding to the causal effects of its parents + bias. In this sense, we want to create a matrix to encode this matrix in such a way that:

$` x_{\text{decoded}} = A x `$  

In this case, $`A = \text{CausalEffectsMatrix}`$ is the matrix that stores the causal equations of the SCM.

### CODE REVIEW

**Comment by mschauer** [comment]
Here you are changing the interface? I had a bit of a feeling that the interface wasn't ripe yet, but did not want to bother you yet. But in future, let's be more careful about not breaking the API needlessy. There is also a "conflict", can you check?

**Comment by JorgeLuizFranco** [comment]
> Here you are changing the interface? I had a bit of a feeling that the interface wasn't ripe yet, but did not want to bother you yet. But in future, let's be more careful about not breaking the API needlessy. There is also a "conflict", can you check?

Sorry, @mschauer . I think I don't get what you mean as "interface". Could you clarify? Going to check the conflicts

**Comment by mschauer** [comment]
Maybe this helps? https://www.youtube.com/watch?v=q3qE2nJRuYM

File: test/equations.jl [hunk 1]
```diff
@@ -37,3 +37,5 @@ println("df_generated: ")
 
 
 display(df_generated)
+
+println(scm.CausalEffectsMatrix)
```
Comment (mschauer) [inline test/equations.jl:41]:
```suggestion
```

**Comment by JorgeLuizFranco** [comment]
@mschauer I forgot to ask, but we need to place @JuliaRegistrator register in the commit comment to register the package, right? Also update de Project.toml with this new version.

**Comment by mschauer** [comment]
Yes, can you prepare a minor version bump to the project as pull request?

**Comment by JorgeLuizFranco** [comment]
> Yes, can you prepare a minor version bump to the project as pull request?

Sure, @mschauer I can prepare it. Which number should I insert in version? 16.0.1?

**Pull request merged**